tests/Oryx.Integration.Tests/LocalDockerTests/NotifyEndOfIntegrationTests.cs

```diff
@@ -0,0 +1,27 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using System.IO;
+using Microsoft.Oryx.Tests.Common;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Oryx.Integration.Tests.LocalDockerTests
+{
+    [Trait("Category", "END")]
+    public class NotifyEndOfIntegrationTests : PlatformEndToEndTestsBase
```

Comment:
What's the purpose of this? Seems a bit "hacky" - maybe add some clarifying documentation?